swb/crop_evapotranspiration.py

```diff
@@ -37,6 +37,8 @@ def _calculate_kc_for_dev_stage(self):
         start = self.planting_date + dt.timedelta(days=self.init)
         end = start + dt.timedelta(days=self.dev - 1)
         dev_kcs = np.linspace(self.kc_ini, self.kc_mid, num=self.dev + 1)[1:]
+        num = len(self.timeseries.loc[start:end, "kc"])
```

Comment:
Please rename `num` variable to a more descriptive name, for example, `total_period_steps` or `period_len`